### PR TITLE
Command to run container on a RHEL docker host

### PR DESCRIPTION
**Pull request by adityai**
Updated README.md to include a docker run command that works on a redhat docker host.

Running the container on a Redhat docker host was throwing and oci error. The comments in the github issue #2155 suggested a slightly different docker command that worked when it was executed on an RHEL7 docker host.

### CODE REVIEW

**Comment by adityai** [comment]
@googlebot I signed it!

**Comment by adityai** [comment]
There is a failure shown in the PR dashboard for 'pull-cadvisor-e2e'. In this PR, only the README.md has been edited.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ sudo docker run \
   gcr.io/cadvisor/cadvisor:$VERSION
 ```
 
+**Note**: For a redhat docker host the above command throws oci errors. Please use the command below if the host is redhat:
```
Comment (iwankgb) [inline README.md:29]: Can you:
- Provide information on RHEL version that is affected
- Improve letter case (RedHat, Docker etc)
- Move information to docs/running.md where some information about RHEL and Centos is already provided.

**Review by iwankgb — CHANGES_REQUESTED** [review]

**Comment by bobbypage** [comment]
> There is a failure shown in the PR dashboard for 'pull-cadvisor-e2e'. In this PR, only the README.md has been edited.

Please rebase, this should be fixed after  #2777, #2778, #2779

File: README.md (hunk 1 above)
Comment (adityai) [reply README.md:29]: will do, thank you, and then I will open a ticket as well - the memory dial stays at zero when the docker host is RedHat.

Comment (adityai) [reply README.md:29]: I followed @bobbypage 's advice in the next comment and did a git pull from google/cadvisor and pushed the latest to adityai/advisor. How do I get the scan or test to run again?

Comment (adityai) [reply README.md]: @iwankgb - I updated as suggested - RHEL version 7, improved letter case RedHat and Docker and I moved the command from readme.md to docs\running.md. Please review.

**Review by iwankgb — CHANGES_REQUESTED** [review]
LGTM. @bobbypage can you confirm, please.

**Review by bobbypage — APPROVED** [review]

**Pull request merged**